python/vitables/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                vitables
+version             3.0.3
+categories          python science
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3
+
+description         ViTables is a graphical tool for PyTables.
+
+long_description    ViTables is a graphical browser for HDF5 and PyTables files. It allows you to \
+                    open HDF5 files, browse their contents and display datasets.
+
+homepage            https://github.com/uvemas/ViTables
+master_sites        pypi:v/vitables
+
+checksums           sha256  24570963c1a2fc8df3ecb39a736256a5384aff114b56540c6bdad7c4197881a0 \
+                    rmd160  f3f66b119afa25b018830f1d10d8c176e44164e3 \
+                    size    871792
+
+python.default_version  312
+
+depends_build-append    port:py${python.version}-setuptools \
+                        port:py${python.version}-hatchling
```

Comment:
please add  `python.pep517_backend hatch` under the `python.default_version`

that will automatically add the `py-hatchling` dependency, and I don't see an mention of `setuptools` - did you verify that this is needed?

Reply:
I have verified and found that `setuptools` should be a run-time dependency, for plugins. Fixed.